src/renderers/dom/client/utils/DOMChildrenOperations.js

```diff
@@ -247,6 +248,39 @@ var DOMChildrenOperations = {
     }
   },
 
+  startObservation: function() { 
+    var config = { attributes: false, childList: true, characterData: false, subtree: true };
+    var MutationObserver = window.MutationObserver || function() {};
+    MutationObserver.prototype.observe = MutationObserver.prototype.observe || function() {};      
+    MutationObserver.prototype.disconnect = MutationObserver.prototype.disconnect || function() {};      
```

Comment:
Instead of using these inline fallbacks, can we just wrap this entire function body in an `if` block that checks for `MutationObserver`? That way it's just a no-op if it doesn't exist.

```js
startObservation: function() { 
  if (MutationObserver) {
  // ...
  }
}
```